### PR TITLE
Integration of pullapprove

### PR DESCRIPTION
**Pull request by AndyScherzinger**
This is the PR for integration https://pullapprove.com

It demands:
* Any PR to be reviewed by at least 2 devs
* Any PR labeled with design to be reviewed by at least 1 designer (Jan, Felix)
* Approval can be approved via commenting: :shipit:, 👍, Approved or LGTM
* Pushes to the PR resets the review process for the code review
* Reopening of a PR resets the review process
* Authors can't approve their own PR

### CODE REVIEW

**Comment by mario** [comment]
Will this mean that if I make a positive review, and someone pushes new things my review won't get reset? Because I think it should.

**Comment by AndyScherzinger** [comment]
at the moment it doesn't but we can activate that. The reason why I didn't activate it is, that I don't know what happens if the PR gets rebased, because then the commits change (hash wise) but that shouldn't reset the review.

**Comment by mario** [comment]
You can still end up in trouble even if just rebasing :)

**Comment by AndyScherzinger** [comment]
SOLD! I activated the review reset for the code review while the designer review won't be reset. This way we don't overdo it and as devs we will make sure that a designer's approval is still valid in case of further code pushes.

File: .pullapprove.yml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  design-review:
+    conditions:
+      labels:
+      - design
+    reset_on_push:
+      enabled: false
+    required: 1
+    users:
+      - jancborchardt 
+      - eppfel
```
Comment (mario) [inline .pullapprove.yml]: Add a newline.

Comment (mario) [inline .pullapprove.yml]: Add a newline here please.

**Comment by AndyScherzinger** [comment]
Thanks @mario ❤️

**Comment by tobiasKaminsky** [comment]
Let us see what happen 👍

**Comment by AndyScherzinger** [comment]
see regex, the 👍 must be at the beginning of a line ;)

**Comment by AndyScherzinger** [comment]
@tobiasKaminsky looks good :)
2017-01-29 20_56_31-nextcloud_android

**Comment by mario** [comment]
@AndyScherzinger please do this for library as well, minus the designers.

**Comment by AndyScherzinger** [comment]
https://github.com/nextcloud/android-library/pull/32

**Comment by AndyScherzinger** [comment]
Already did ;)

**Comment by AndyScherzinger** [comment]
cc: @MorrisJobke @LukasReschke we replaced LGTM with pullapprove which works an is more flexible :)

@jancborchardt @eppfel the plugin now enforces an approval for all PRs labeled with design :)

**Pull request merged**